packages/backend/src/plugins/catalog.ts

```diff
@@ -1,14 +1,27 @@
 import { CatalogBuilder } from '@backstage/plugin-catalog-backend';
 import { ScaffolderEntitiesProcessor } from '@backstage/plugin-scaffolder-backend';
+import { ManagedClusterProvider } from '@janus-idp/backstage-plugin-ocm-backend';
 import { Router } from 'express';
 import { PluginEnvironment } from '../types';
 
 export default async function createPlugin(
   env: PluginEnvironment,
 ): Promise<Router> {
   const builder = await CatalogBuilder.create(env);
+  const ocm = ManagedClusterProvider.fromConfig(env.config, {
+    logger: env.logger,
+  });
+  builder.addEntityProvider(ocm);
   builder.addProcessor(new ScaffolderEntitiesProcessor());
   const { processingEngine, router } = await builder.build();
   await processingEngine.start();
+  await env.scheduler.scheduleTask({
+    id: 'run_ocm_refresh',
+    fn: async () => {
+      await ocm.run();
+    },
+    frequency: { minutes: 30 },
+    timeout: { minutes: 10 },
```

Comment:
Can we add this in the app-config.yaml?

Reply:
I don't think we can put this into the `app-config.yaml` at this time, this would require some additional changes to the source code. However we can talk about this more and maybe implement it in the future.

Reply:
no problem. I just wanted to keep track here.
Will merge later today. Need the environment up for a recording.